Application/EdFi.Ods.WebApi/appsettings.json

```diff
@@ -47,7 +47,7 @@
       },
       {
         "Name": "Profiles",
-        "IsEnabled": false
+        "IsEnabled": true
```

Comment:
We're probably ok to have Profiles enabled by default, since it won't really do much if there are no profiles in the solution. Let's get @vimayya to confirm.

Reply:
I think it is ok now that we have removed the sample profiles. Otherwise I would have wanted it disabled to keep parity with the cloud binaries.